### DISCUSSION TITLE
How do I add Google Tag Manager code to inside <body> tag?

### DISCUSSION THREAD

**Discussion by vovavasylyna** [imfing/hextra#257] [Q&A] [answered]
Hello, how do I add Google Tag Manager code to inside body tag? 
I managed to add the script inside head tag. But how do I add noscript inside body, in fact right after the opening body tag?

**Comment by imfing** [comment, marked as answer]
You need to override the default template and put the tags there

https://github.com/imfing/hextra/blob/main/layouts/_default/baseof.html
